addons/account/models/account_move.py

```diff
@@ -2075,6 +2075,8 @@ def is_internal_partner(partner):
         return move
 
     def post(self):
+        if not self.env.su and not self.env.user.has_group('account.group_account_invoice'):
```

Comment:
maybe add a comment for this strange check?
``` python
# `user_has_group` won't be bypassed by `sudo` since it doesn't change the user anymore.
```